source/_docs/wordpress-export.md

```diff
@@ -5,15 +5,9 @@ tags: [export]
 categories: [wordpress]
 ---
 
-Follow the recommended process within [Migrate to Pantheon: WordPress](/docs/migrate-wordpress), which uses the Pantheon Migration plugin.
-
-The following scenarios are exceptions to the recommended process and require [manually migrating](https://pantheon.io/docs/migrate-manual/) the site:
```

Comment:
I definitely like avoiding dupe content, but in this case it helps people determine whether or not they're headed down the right track. Folks may land here and do unnecessary work, only to later learn they could have migrated with a few mouse clicks in the guided process.

Consolidating/simplifying our migration docs may be the ultimate solution, but that seems like work for another issue.